readthedocs/projects/models.py

```diff
@@ -213,10 +213,24 @@ class Project(models.Model):
         ),
     )
 
+    # External versions
     external_builds_enabled = models.BooleanField(
         _('Build pull requests for this project'),
         default=False,
-        help_text=_('More information in <a href="https://docs.readthedocs.io/en/latest/guides/autobuild-docs-for-pull-requests.html">our docs</a>')  # noqa
+        help_text=_('More information in <a href="https://docs.readthedocs.io/page/guides/autobuild-docs-for-pull-requests.html">our docs</a>')  # noqa
+    )
+    external_builds_privacy_level = models.CharField(
+        _('Privacy level of builds from pull requests'),
+        max_length=20,
+        # TODO: remove after migration
+        null=True,
+        choices=constants.PRIVACY_CHOICES,
+        # Force it to private so current projects in .com
+        # use this value.
+        default=constants.PRIVATE,
```

Comment:
Shouldn't this default to `settings.DEFAUL_PRIVACY_LEVEL` so we have `public` in .org and `private` in .com? We should update the migration in that case, to not hardcode the string.

Reply:
The migration will evaluate the current value when the migration was created, it won't use the constant. And when adding the default value to existing projects it will use the value from the migration. This can be changed after the migration.